charts/gardener-dashboard/templates/configmap.yaml

```diff
@@ -300,4 +300,17 @@ data:
 {{ toYaml . | trim | indent 12 }}
           {{- end }}
         {{- end }}
+      {{- end }}
+      {{- if .Values.frontendConfig.vendorHints }}
+      vendorHints:
+        {{- range .Values.frontendConfig.vendorHints }}
+        - vendorNames:
+            {{- range .vendorNames }}
+            - {{ . }}
+            {{- end }}
+          hintMessage: {{ .hintMessage }}
```

Comment:
May be it makes to change it to {{ quote .hintMessage }}

Reply:
yes, I already did this and added the support for markdown on the UI side as well